roles/satellite-clone/tasks/main.yml

```diff
@@ -115,6 +115,12 @@
     name: "{{ satellite_package }}"
     state: latest
 
+# due to support for remote databases in 6.4 the postgres user (and server) may be missing
```

Comment:
Mongodb can also be external right, any reason why it is not checked explicitly here like postgres?

Reply:
That hadn't occurred to me at the time :(